packages/react/time-series/src/ErrorFallback.tsx

```diff
@@ -15,19 +15,37 @@ export interface ErrorFallbackProps {
 export function ErrorFallback(props: ErrorFallbackProps) {
   const { error = serverErrorMessage, styles } = props
 
-  const width = styles?.canvas?.width
-  const height = styles?.canvas?.height || defaultChartHeight
+  const [isSmall, setIsSmall] = React.useState(false)
+
+  const containerRef = React.useRef<HTMLDivElement>(null)
+
+  const canvasWidth = styles?.canvas?.width
+  const canvasHeight = styles?.canvas?.height || defaultChartHeight
+
+  React.useLayoutEffect(() => {
+    const containerWidth = containerRef.current?.clientWidth
+    const containerHeight = containerRef.current?.clientHeight
+
+    const isContainerSmall = (containerWidth && containerWidth < 150) || (containerHeight && containerHeight < 150)
+    const isCanvasSmall = (canvasWidth && canvasWidth < 150) || canvasHeight < 150
```

Comment:
What about adding those `150` to our defaults?

Reply:
I think that would be good, I'm adding it